src/v/redpanda/admin_server.cc

```diff
@@ -1004,6 +1004,21 @@ void admin_server::register_cluster_config_routes() {
               }
           }
 
+          if (get_boolean_query_param(*req, "dry_run")) {
```

Comment:
should we do the validation above even if `force` is true (ie `url?force=true&dry_run=true`):

```c++
const auto dry_run = get_boolean_query_param(*req, "dry_run");
if (!get_boolean_query_param(*req, "force") || dry_run) {
   //validate
   ...

if (dry_run) {
   ...
   co_return
```

Reply:
I prefer to take the flags at face value: force means skip validate, dry_run means skip apply, so the combination of both means do nothing.